swap_meet/clothing.py

```diff
@@ -1,2 +1,38 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    # May have to change fabric default values to None
```

Comment:
You didn't have to do it for the string `"Unknown"` because strings are immutable and it's okay for the same instance of a string to be shared. Same with integers. Only the empty list (`[]`) ends up being a problem in the swap meet project.